src/components/organisms/EntityDetails/EntityDetailsContent/Settings/SettingsScheduling/Schedule.tsx

```diff
@@ -136,6 +136,12 @@ const Schedule: React.FC = () => {
     }
   }, [cronString]);
 
+  const selectTemplateValue = useMemo(() => {
+    const value = quickOptions.find(option => option.value === templateValue)?.value;
+
+    return value === undefined ? custom.value : value;
```

Comment:
`templateValue` is set internally based on `quickOptions`, so there will be always option found.

Because of that, this is code is equivalent of:

```suggestion
    return templateValue === undefined ? custom.value : value;
```

or:

```suggestion
    return templateValue ?? custom.value;
```

Reply:
Oh, yes, ?? works here, thanks